nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/SessionImpl.java

```diff
@@ -976,6 +979,10 @@ public Node addProxy(Serializable targetId, Serializable versionableId, Node par
         Node proxy = addChildNode(parent, name, pos, Model.PROXY_TYPE, false);
         proxy.setSimpleProperty(Model.PROXY_TARGET_PROP, targetId);
         proxy.setSimpleProperty(Model.PROXY_VERSIONABLE_PROP, versionableId);
+        if (changeTokenEnabled) {
+            proxy.setSimpleProperty(Model.MAIN_SYS_CHANGE_TOKEN_PROP, Model.INITIAL_SYS_CHANGE_TOKEN);
+            proxy.setSimpleProperty(Model.MAIN_CHANGE_TOKEN_PROP, Model.INITIAL_CHANGE_TOKEN);
```

Comment:
We initialize the values to 0 at creation time. Proxies can have user-modified proxy-specific schemas (in theory).